src/main/java/com/notification/model/request/PasswordResetTokenRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.notification.model.request;
+
+import com.notification.dto.User;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class PasswordResetTokenRequest {
+    String token;
+    User user;
```

Comment:
use fullName and sentToEmail instead of user class.

Reply:
Resolved